### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.5 - es. 3.9 (21 ottobre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#296] [Esercizi M-Z]
<img width="954" alt="immagine" src="https://github.com/user-attachments/assets/417d5f50-a8cc-4dfa-bec7-1be553e14908">

**Comment by Andrii04** [comment]
![IMG_0718](https://github.com/user-attachments/assets/eec38407-5b0a-4ffb-82b5-f2c2f0d3f3a7)

**Reply by scuty2000** [reply]
Attento, qui quando hai fatto P(Fb|Ab) hai considerato solo il caso in cui si estrae una pallina bianca da B ed una rossa da C, ma va considerato anche il caso in cui estrai una rossa da B ed una bianca da C

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/dc634a9c-e9e5-4a64-a5ba-4d776d28f5d9)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/0f805a60-d3df-4ecd-97c9-73e628818214)
